os/kernel/binary_manager/binary_manager.c

```diff
@@ -197,4 +205,20 @@ int binary_manager(int argc, char *argv[])
 	}
 
 	return 0;
+errout_with_nobinary:
+	while (1) {
+		lldbg("=============== !!ERROR!! ============== \n");
+#ifdef CONFIG_APP_BINARY_SEPARATION
+		if (!is_found_ubin) {
+			lldbg("ERROR!! Not found valid user binaries.\n");
+			lldbg("Please check user binary and smartfs configurations.\n");
+		}
+#endif
+		if (!is_found_kbin) {
+			lldbg("ERROR!! Not found kernel partitions.\n");
+			lldbg("Please check kernel partition configurations.\n");
+		}
+		lldbg("Enable CONFIG_DEBUG_BINMGR_ERROR if you want to see the logs for failures.\n\n");
```

Comment:
In the above, it says that there is no valid binary or partition. That is the log for failure.
So I think `if you want to see the logs for failures` can be changed to include more details.